packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -0,0 +1,134 @@
+  insertStoryTitle,
+  publishStory,
+  addRequestInterception,
+} from '@web-stories-wp/e2e-test-utils';
+
+async function createStoryWithTitle(title) {
+  await createNewStory();
+  await insertStoryTitle(title);
+  await expect(page).toClick('#library-tab-text');
+  const insertButton = await page.waitForXPath(
+    `//button//span[contains(text(), 'Title 1')]`
+  );
+  await insertButton.click();
+  await expect(page).toMatchElement('[data-testid="textFrame"]', {
+    text: 'Title 1',
+  });
+}
+
+async function updateFont(fontFamily) {
+  // Open style pane
+  await expect(page).toClick('li[role="tab"]', { text: /^Style$/ });
+  await expect(page).toClick('button[aria-label="Font family"]');
+  await page.keyboard.type(fontFamily);
+  await page.keyboard.press('ArrowDown');
+  await page.keyboard.press('Enter');
+}
+
+async function addStoryWithFont(title) {
+  await createStoryWithTitle(title);
+  await updateFont('Rock Salt');
+  await publishStory();
+}
+
+describe('Font Check Metrics', () => {
+  let stopRequestInterception;
+  let mockResponse;
+
+  beforeAll(async () => {
+    await page.setRequestInterception(true);
+    stopRequestInterception = addRequestInterception((request) => {
+      if (request.url().includes('/v1/fonts/') && mockResponse) {
+        request.respond(mockResponse);
+        return;
+      }
+
+      request.continue();
+    });
+  });
+
+  afterAll(async () => {
+    await page.setRequestInterception(false);
+    stopRequestInterception();
+  });
+
+  //
+
+  it('should receive updated font metrics', async () => {
+    mockResponse = null;
+    await addStoryWithFont('Font Check Metrics');
+    const textElBefore = await expect(page).toMatchElement('p', {
+      text: 'Title 1',
+    });
+    const fontFamilyBefore = await page.evaluate(
+      (el) => getComputedStyle(el).font,
+      textElBefore
+    );
+    expect(fontFamilyBefore).toContain('Rock Salt');
+    expect(fontFamilyBefore).toContain('cursive');
+    mockResponse = [
+      {
+        family: 'Rock Salt',
+        fallbacks: ['sans-serif'],
+        weights: [900],
+        styles: ['italic'],
+        variants: [[0, 400]],
+        service: 'fonts.google.com',
+        metrics: {
+          upm: 200,
+          asc: 1623,
+          des: -788,
+          tAsc: 824,
+          tDes: -240,
+          tLGap: 63,
+          wAsc: 1623,
+          wDes: 788,
+          xH: 833,
+          capH: 1154,
+          yMin: -787,
+          yMax: 1623,
+          hAsc: 1623,
+          hDes: -788,
+          lGap: 32,
+        },
+      },
+    ];
+
+    await page.reload({ waitUntil: ['networkidle0', 'domcontentloaded'] });
+    const textElAfter = await expect(page).toMatchElement('p', {
+      text: 'Title 1',
+    });
+    const fontFamilyAfter = await page.evaluate(
+      (el) => getComputedStyle(el).font,
+      textElAfter
+    );
+    expect(fontFamilyAfter).toContain('Rock Salt');
+    //expect(fontFamilyAfter).toContain("sans-serif");
```

Comment:
Intercepting the request with the updated metrics isn't quite working for this test --- for sure we can check the style / font family (in the dom) to see if the updates were applied --- might look at trying to use the dev tools for that part.